frontend/app/components/auth-panel/auth-panel.tsx

```diff
@@ -522,7 +324,20 @@ function getSortArray(currentSort: Sorting, intl: IntlShape) {
   });
 }
 
-export const AuthPanelWithIntl = (props: PropsWithoutIntl) => {
+export default function(props: OwnProps) {
```

Comment:
I recommend to use named function. It is better for development, you will see function name in call stack and react dom tree.

Reply:
```suggestion
export default function AuthPanelConnected(props: OwnProps) {
```

Reply:
Yes it is much better. But I prefer to avoid default export, because  it is potential buggy approach.